tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaHelper.cs

```diff
@@ -225,6 +230,11 @@ private static long GetMessageSize<T>(T message)
                     tags.Tombstone = "true";
                 }
 
+                if (!string.IsNullOrEmpty(topic))
+                {
+                    span.SetTag(Tags.MessagingDestinationName, topic);
```

Comment:
Instead of `span.SetTag` modifying `tags` directly would be better, but refer to my other comment on a more performant / common format we use for tags